pkg/gadgets/common/mntns_filter.h

```diff
@@ -0,0 +1,33 @@
+/* SPDX-License-Identifier: (GPL-2.0 WITH Linux-syscall-note) OR Apache-2.0 */
+
+#ifndef MNTNS_FILTER_H
+#define MNTNS_FILTER_H
+
+#include <bpf/bpf_helpers.h>
+#include <bpf/bpf_core_read.h>
+#include <bpf/bpf_tracing.h>
+
+const volatile bool filter_by_mnt_ns = false;
+
+struct {
+	__uint(type, BPF_MAP_TYPE_HASH);
+	__type(key, __u64);
+	__type(value, __u32);
+	__uint(max_entries, 1024);
+} mount_ns_filter SEC(".maps");
+
+static __always_inline bool should_filter_mntns_id(__u64 mntns_id) {
```

Comment:
Just a general question about the wording with `filter`. Does this mean it gets filtered out or filtered so it stays -> we want to have it.

(I know that this implementation is for filtering out)

Reply:
I also have doubts regarding this. It's called `filter` from filter out, but I'm wondering if there is a more clear terminology to use.